assets/src/api.ts

```diff
@@ -154,9 +154,9 @@ export const fetchNearestIntersection = (
     defaultResult: null,
   })
 
-export const fetchSwings = (): Promise<Swing[] | null> =>
+export const fetchSwings = (routeIds: RouteId[]): Promise<Swing[] | null> =>
   apiCall({
-    url: "/api/swings",
+    url: `/api/swings?route_ids=${routeIds.join(",")}`,
```

Comment:
**thought:** Given that our route IDs are pretty short, this probably isn't a concern in the real world, but theoretically could this be a problem if the number of routes was long enough that we exceeded the URL length "limit"?

Reply:
Hmm, I don't think that's much of a concern immediately but I could see that being an issue for our endpoints in general. I'll definitely keep it in mind.